core-common/src/main/java/org/apache/kylin/common/KylinConfigBase.java

```diff
@@ -982,6 +982,14 @@ public int getDefaultDecimalScale() {
         }
     }
 
+    public boolean enableHiveDdlQuote(){
+        return Boolean.parseBoolean(getOptional("kylin.source.hive.quote-enabled", TRUE));
+    }
+
+    public static String getQuoteCharacter(){
```

Comment:
Why not make this configurable?

Reply:
The quote character used by Hive is **`**, I think it wouldn't be changed easily.

Reply:
Make it configurable do not make harm, I will change it.